pkg/providers/aws/provider_postgres.go

```diff
@@ -1157,8 +1131,7 @@ func (p *PostgresProvider) setPostgresDeletionTimestampMetric(ctx context.Contex
 			logrus.Errorf("failed to get cluster id while exposing information metric for %v", instanceName)
 			return
 		}
-
-		labels := buildPostgresStatusMetricsLabels(cr, clusterID, instanceName, cr.Status.Phase)
+		labels := resources.BuildStatusMetricsLabels(cr.ObjectMeta, clusterID, instanceName, postgresProviderName, cr.Status.Phase)
```

Comment:
Code coverage is failing because you touched this line , it may have never been covered but because you change it , its triggering codecov.

Reply:
Just checked master looks like it was never covered

Reply:
Added a couple of test cases for it 👍🏻